src/ORM/Association.php

```diff
@@ -1390,6 +1390,7 @@ public function __call($method, $argument)
      * - foreignKey: The name of the field used to relate both tables
      * - conditions: List of conditions to be passed to the query where() method
      * - sort: The direction in which the records should be returned
+     * - sortOverwrite: (default true) True to replace configured sort order, false to merge.
```

Comment:
From reading this, I'm not sure when this option takes effect or what is scope is.